package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "fluid.js",
```

Comment:
The package name is `fluid-canvas`, so the `package-lock.json` should match the `package.json`. (This was done automatically by `npm`).